src/components/Personalization/flicker/index.js

```diff
@@ -38,3 +46,37 @@ export const showElements = prehidingSelector => {
     removeNode(node);
   }
```

Comment:
Should you delete the element from the CACHE too?  If you don't do this, calling hideElements with the same selector will not work.

Reply:
good catch, added `delete CACHE[prehidingSelector]`